internal/service/cluster/cluster.go

```
@@ -291,7 +294,8 @@ func (s *Service) Run(ctx context.Context, host service.Host) error {
 
 		err := s.node.Start(nil)
```

I wonder if we should do that if our policy is to fail fast and fail hard on configuration issues and prevent split-brain.

---

cc @tpaschalis if you have context on why we do this